test/Gridify.Tests/IssueTests/Issue286Tests.cs

```diff
@@ -0,0 +1,34 @@
+
+using System.Linq;
+using AutoBogus;
+using Xunit;
+
+namespace Gridify.Tests.IssueTests;
+
+public class TestClassGridifyMapper : GridifyMapper<TestClass>
+{
+   public TestClassGridifyMapper()
+   {
+      GenerateMappings()
+         .AddMap("ChildName", e => e.ChildClass.Name);
+   }
+}
+
+public class Issue286Tests
+{
+   [Fact]
+   public void Mapping_ShouldAllowNullProperty()
+   {
+      GridifyGlobalConfiguration.AvoidNullReference = true;
```

Comment:
Use GridifyMapper configuration instead, this would cause concurrency issues in the tests. related to #179